pom.xml

```diff
@@ -1075,15 +1045,6 @@
         <version>${version.com.jcraft}</version>
       </dependency>
 
-      <!-- Required to run the ElasticSearch server when performing the unit test executions.
-          The ELS data provider requires groovy scripts enabled on the server.-->
-      <dependency>
-        <groupId>org.codehaus.groovy</groupId>
-        <artifactId>groovy-all</artifactId>
-        <classifier>indy</classifier>
-        <version>${version.org.codehaus.groovy}</version>
-      </dependency>
-
```

Comment:
it seems this is needed in kie-soup?
```
[2022-10-07T09:53:01.927Z] [ERROR] 'dependencies.dependency.version' for org.codehaus.groovy:groovy-all:jar:indy must be a valid version but is '${version.org.codehaus.groovy}'. @ org.kie.soup:kie-soup-parent:7.74.0-SNAPSHOT, /home/jenkins/workspace/KIE/main/pullrequest/drools-wb-main.pr/bc/kiegroup_kie_soup/pom.xml, line 498, column 18
[2022-10-07T09:53:01.927Z] 
```

Reply:
that could be part of Dashbuilder @jesuino